generators/client/templates/vue/src/main/webapp/app/core/jhi-navbar/jhi-navbar.component.ts.ejs

```diff
@@ -105,4 +107,18 @@ export default class JhiNavbar extends Vue {
   public get inProduction() : boolean {
     return (this.$store.getters.activeProfiles.indexOf('prod') > -1);
   }
+
+<%_ if (enableTranslation) { _%>
+  findLanguage(): string {
+    const languageKeys = Object.keys(JSON.parse(JSON.stringify(this.languages)));
```

Comment:
why is JSON parse/stringify here, languages is already a proper object

Reply:
I dont think this is right, with this logic you could end up with wrong versions for many languages, I'll rather do just the below

```
const currentLanguage = Object.keys(this.languages).includes(navigator.language) ? navigator.language : this.currentLanguage;
this.translationService().refreshTranslation(currentLanguage);
```

Reply:
Thanks for your feedback. I test your code and the outcome is the same for our test cases. I have changed the code in the branch last Friday. I don't know whether that is the right way to update a pull request or not.